src/tietze.c

```diff
@@ -240,6 +240,7 @@ Obj FuncTzSortC (
         SET_LEN_PLIST( rels,  i );  SHRINK_PLIST( rels,  i );
         SET_LEN_PLIST( lens,  i );  SHRINK_PLIST( lens,  i );
         SET_LEN_PLIST( flags, i );  SHRINK_PLIST( flags, i );
+        ptTietze = ADDR_OBJ( tietze );
         ptTietze[TZ_NUMRELS] = INTOBJ_INT(i);
```

Comment:
Note that `ptTietze` is a plist, so perhaps instead replace both lines by `SET_ELM_PLIST(ptTietze, TZ_NUMRELS, INTOBJ_INT(i));`?

Reply:
That's a good point. In general, there seems to be far too much storing of pointers in this file, for (I assume) efficiency. Keeping them in the odd tight loop is fine, but I'd prefer to be proactive (grab the pointer just before you need it) rather than reactive (try to regrab the pointer whenever you might break it, which is easy to get wrong).